src/docs/operator-guides/how-to-edit-pages/index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: How to edit content pages in Console
+slug: how-to-edit-content-pages-in-console
+updated: 2023-02-14
+category: operator-guides
+ingress:
+  Learn how to use Pages to edit content across Landing page,
+  informational pages, or blog pages.
+published: true
+---
+
+This guide introduces how to use Pages. Pages lets you edit your
+marketplace’s static content pages, like your Landing page or About
+page. If you would like more background on the Pages feature, consult
+this [Introduction to Pages article](https://www.sharetribe.com/docs/operator-guides/introducing-pages/).
+
+## Introduction
+
+### Gettings started
+
+Find the Pages feature in [Console.](https://flex-console.sharetribe.com/) Navigate to your Development environment, pick the “Build” section from your top bar, select “Content” from the sub-menu, then select “Pages” from the left side menu. You should now see your default content pages (the About page, the Landing page, the Terms of use page, and the Policy page) ready for editing. 
+
+If you don't have the feature available in Console and want to start
+using it, check out our article about
+[enabling the Pages feature in your marketplace](https://www.sharetribe.com/docs/operator-guides/how-to-enable-pages/#for-marketplaces-created-before-2023-02-14-pages-release-date).
+
+### Sections, Blocks, Fields, and Section templates
+
+Content pages in your marketplace are built using Sections, Blocks, and
+Fields.
+[Section templates](https://www.sharetribe.com/docs/operator-guides/section-templates)
+determine layout and content type per Section.
+
+Any Page, including your Landing page, consists of a set of Sections. A
+Section is a part of the page that takes full page width and has
+predetermined content inside it, in a specific layout, determined by the
+chosen Section template. Every Section in a content page is created and
+managed independently from other Sections.
+
+The basic content of the section is stored using fields. Examples of
+Section fields include title, description, button content, and
+background.
+
+Each Section can also contain any number of Blocks. Blocks normally
+contain more detailed information for a Section. Each Block is made up
+of different fields like image, videos, titles, ingresses, texts, and
+buttons.
+
+The default landing page in a Flex marketplace Demo environment has
+different type of Sections that will help you get an idea of what is
+possible with the Pages editor.
```

Comment:
"Pages editor" is not a part of our customer-facing terminology. We say just "Pages". Internally we talk about "Page Manager" and "Page Editor" to separate the view that lists pages and the view that edits them, but I don't think we need these two externally. Just "Pages" instead.